coding/coding_tests/CMakeLists.txt

```diff
@@ -49,4 +50,4 @@ set(
 
 omim_add_test(${PROJECT_NAME} ${SRC})
 
-omim_link_libraries(${PROJECT_NAME} coding base geometry minizip succinct ${LIBZ})
+omim_link_libraries(${PROJECT_NAME} coding base geometry minizip succinct stats_client platform platform_tests_support ${Qt5Widgets_LIBRARIES} ${LIBZ})
```

Comment:
platform uses coding, platform_tests_support uses platform. Please, fix the link order.